src/tests/auth/userVerification.test.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server';
+import { user } from '../fixtures/user.fixture';
+import TokenService from '../../services/token.service';
+
+const token = TokenService.generateToken({
```

Comment:
put this variable in the `fixtures` folder.

And if the file name has 2 words use `-` instead of `camelCase`. For example, rename the file to `user-verification.test.js`